ansible/roles/adm/tasks/main.yml

```diff
@@ -39,3 +39,17 @@
 
 - name: sudoers.d/adm
   template: src=sudoers dest=/etc/sudoers.d/adm owner=root group=root mode=0440 validate='visudo -cf %s'
+
+- name: lockdown access to ssh
+  lineinfile: dest=/etc/ssh/sshd_config
+              regexp="^PasswordAuthentication"
+              line="PasswordAuthentication no"
+              state=present
+  notify: restart ssh
+
+- name: do not allow root to login via ssh
+  lineinfile: dest=/etc/ssh/sshd_config
+              regexp="^PermitRootLogin"
+              line="PermitRootLogin no"
+              state=present
+  notify: restart ssh
```

Comment:
IDK. Maybe `dom0` as this role will accumulate all "ooniized" defaults for host system, including mentioned at #101.